configs.py

```diff
@@ -94,8 +127,9 @@ class SmokeConfig(BaseConfig):
     hostname: str = ""
     sweep_config: str = ""
     data_source: DataSourceEnum = DataSourceEnum.MiniMAD
+    base_model_name: BaseModelEnum = BaseModelEnum.RNN
 
-    model_config: ViTConfig = ViTConfig(patch_size=8, depth=1, heads=1, mlp_dim=4)
+    base_model: BASE_MODEL_CONFIG_CLASS[base_model_name] = BASE_MODEL_SMOKE_CONFIG[base_model_name]
```

Comment:
how does this even work, i don't think it works, it's just ignored

Reply:
maybe, tho it doesn't work when not putting a type at all